src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,13 +44,12 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import javax.xml.ws.http.HTTPException;
 import org.json.simple.JSONObject;
 
 /**
  * Servlet responsible for storing Youtube Video and Displaying the details of the Youtube Video
  */
-@WebServlet("/data")
+@WebServlet("/data1")
```

Comment:
nit: can you revert this back to `/data`?

If the endpoint is stale, try to delete the `targets/` folder, or feel free to reach out to VSEs to debug further, but I don't think this is the long term solution, so we shouldn't be committing this.

However, if you want to rename the endpoint, I think it is a good idea. Can we pick a more descriptive name? For example, `/videos-sentiment`?